resources/core/charts/gateway/templates/gateway.yaml

```diff
@@ -16,4 +16,13 @@ spec:
       serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
       privateKey: /etc/istio/ingressgateway-certs/tls.key
     hosts:
-    - "*.{{.Values.global.domainName}}"
\ No newline at end of file
+    - "*.{{.Values.global.domainName}}"
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    tls:
+      httpsRedirect: true # automatic 301 redirect from http to https
+    hosts:
+    - "console.{{.Values.global.domainName}}"
```

Comment:
Instead of listing hosts here please provide a wildcard like we have above.
If we have console and grafana we could have as well dex, jaeger, ui-api etc. so it's better have it like with https